sdk/python/kfp/dsl/_container_op.py

```diff
@@ -72,11 +72,15 @@ def _proxy_container_op_props(cls: "ContainerOp"):
     return cls
 
 
-def as_list(value: Any, if_none: Union[None, List] = None) -> List:
+def as_string_list(list_or_str: Union[Any, Sequence[Any]], if_none: Optional[List] = None) -> List[str]:
```

Comment:
Let's change if_none to default?

Reply:
> Let's change if_none to default?

I totally agree that `if_none` is a bad name =)
I'll change it since I'm editing this function anyway.

BTW, what do you think about only supporting lists in `command` or `args`?

Reply:
I've just removed `if_none` since it's not used anywhere.